src/game/default/g_main.h

```diff
@@ -34,13 +39,16 @@ extern g_export_t ge;
 
 extern uint32_t g_means_of_death;
 
+extern cvar_t *g_admin_password;
 extern cvar_t *g_ammo_respawn_time;
 extern cvar_t *g_auto_join;
 extern cvar_t *g_capture_limit;
 extern cvar_t *g_cheats;
 extern cvar_t *g_ctf;
 extern cvar_t *g_frag_limit;
 extern cvar_t *g_friendly_fire;
+extern cvar_t *g_force_demo;
```

Comment:
I agree it should be configurable client-side but also enabling it server-side gives admins more control over this and so they can enforce it where necessary (tournaments).